plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLSourceViewer.java

```diff
@@ -124,6 +124,15 @@ protected boolean isReadOnly()
     protected void setSourceText(DBRProgressMonitor monitor, String sourceText)
     {
     }
+    
+    @Override
+    public void activatePart() {
+    	super.activatePart();
+    	
+    	if (!getSourceObject().isPersisted()) {
+    		refreshPart(this, true);
+    	}
```

Comment:
```suggestion
        super.activatePart();
        
        if (!getSourceObject().isPersisted()) {
            refreshPart(this, true);
        }
```
Usage of tabs instead of spaces